src/services/EthereumServiceTesting.ts

```diff
@@ -51,14 +49,10 @@ export class EthereumServiceTesting {
   ]);
 
   private async fireAccountsChangedHandler(account: Address) {
-    if (account && !(await this.disclaimerService.ensurePrimeDisclaimed(account))) {
```

Comment:
- Not super happy about this change
- Reason: On first app load, we have this `EthereumService#connectToConnectedProvider` method, that checks for disclaimer. That hindered testing.
- Change seems minor, and rejecting Disclaimer will still block user --> TODO later: Add disclaimer test